pycbc/workflow/core.py

```diff
@@ -1122,12 +1122,13 @@ def __init__(self, ifos, exe_name, segs, file_url=None,
         exe_name: string
             A short description of the executable description, tagging
             only the program that ran this job.
-        segs : igwn_segments.segment or igwn_segments.segmentlist
+        segs : igwn_segments.segment or igwn_segments.segmentlist or None
```

Comment:
This change is rejected (or at least musn't be bundled in with this patch). Following other workflows, we provide a "dummy" segment if there is not a logical one to give, but a segment should still be given.

Reply:
This has been reverted - it means that the files in the workflow are all named "...-0-0.ext" or similar, but that is a minor thing